__tests__/ga.test.js

```diff
@@ -1,34 +1,34 @@
-const util = require("util");
-const integration = "ga";
-const name = "Google Analytics";
-
 const fs = require("fs");
 const path = require("path");
-const version = "v0";
 
-const transformer = require(`../${version}/destinations/ga/transform`);
+const version = "v0";
+const sources = ["web"]
+const integration = "ga";
+const name = "Google Analytics";
 
-const inputDataFile = fs.readFileSync(
-  path.resolve(__dirname, `./data/${integration}_input.json`)
-);
+const transformer = require(`../${version}/destinations/${integration}/transform`);
 
-const outputDataFile = fs.readFileSync(
-  path.resolve(__dirname, `./data/${integration}_output.json`)
-);
+sources.forEach(source => {
+  const dataFile = fs.readFileSync(
+    path.resolve(__dirname, `./data/${integration}/${source}.json`)
+  );
 
-const inputData = JSON.parse(inputDataFile);
-const expectedData = JSON.parse(outputDataFile);
+  const testData = JSON.parse(dataFile);
 
-inputData.forEach((input, index) => {
-  test(`${name} Tests : payload: ${index}`, () => {
-    let output, expected;
-    try {
-      output = transformer.process(input);
-      expected = expectedData[index];
-    } catch (error) {
-      output = error.message;
-      expected = expectedData[index].message;
-    }
-    expect(output).toEqual(expected);
-  });
+  Object.keys(testData).forEach(key => {
+    test(`${name} Tests : payload: ${key}`, () => {
```

Comment:
let's name the keys like test-1 or something for easier lookup ?
1/2 etc seems too generic. Going forward we can name them around the specific functionality they are for.

also. add source name to jest test name.